plugins/meta/tuning/tuning.go

```diff
@@ -90,6 +164,26 @@ func cmdAdd(args *skel.CmdArgs) error {
 				return err
 			}
 		}
+
+		var err error
+
+		if tuningConf.Mac != "" {
+			if err = changeMacAddr(args.IfName, tuningConf.Mac); err == nil {
+				err = updateResultsMacAddr(*tuningConf, args.IfName, tuningConf.Mac)
```

Comment:
I don't think this `err` is ever passed back.

Reply:
@s1061123 since updateResultsMacAddr() never returns any thing except nil, I think you should just remove the return value from updateResultsMacAddr().

Reply:
I removed 'return' in updateResultsMacAddr(). Thank you all!